app/containers/Forms/FormSharedStyles.tsx

```diff
@@ -33,4 +33,9 @@ export default css.global`
     color: white !important;
     padding: 5px;
   }
+  .exit-button {
+    background: white !important;
```

Comment:
not a huge fan of the `!important`. Maybe an inline style on the `Button` element is a better alternative? This would be a temporary solution until we explore using the bcgov boostrap theme.